nodetypes/radon.nodes.aws/AwsLambdaFunction/files/create/create.yml

```diff
@@ -20,7 +20,7 @@
           --function-name {{ function_name }}
           --reserved-concurrent-executions {{ lambda_concurrency }}
           --region {{ aws_region }}
-      when: (lambda_concurrency is defined) and (lambda_concurrency != 0)
```

Comment:
Have you tested the deployment after this change? ... also when concurrency is disabled? I remember that we had some issues with xOpera whiteout this addition. @v-yussupov what do you think?

Reply:
ping @v-yussupov

Reply:
yes, the ansible task fails if `lambda_concurrency` is absent, which is the case when modelers do not define this optional property in templates.

Reply:
No, I think this looks like a bug of xOpera, which should be able to deal with unspecified optional properties. I will revert this change in the pull request at present, but we need to discuss this issue with XLAB perhaps in the next WP4 call. The decomposition tool doesn't recognize a concurrency of zero, which in my opinion is a bit confusing to the users.

Reply:
Once this is reverted we can merge it. Thanks.